### PR TITLE
Update 1.3.0

### PR DESCRIPTION
**Pull request by YuuuXie**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

@jan-janssen , I tried `conda install mir-flare` and there are a few issues that I got 😔:
1. The compiled flare library is of name `lib_C_flare.so`, while we did not set or use the `lib` prefix, so when I do `import flare.bffs.sgp._C_flare` the module is not found. I checked the doc, and think it is related to the [default setting here](https://conda-forge.org/docs/maintainer/knowledge_base.html#using-cmake) that `CMAKE_INSTALL_LIBDIR=lib` is used by default, so "lib" is put in front of the original "_C_flare.so". I therefore set `CMAKE_INSTALL_LIBDIR` to empty in the cmake.patch
2. After I rename `lib_C_flare.so` to `_C_flare.so` I still get the import issue when doing `import flare.bffs.sgp._C_flare`:
```python
ImportError: /n/home08/xiey/.conda/envs/flaretest/lib/python3.8/site-packages/flare/bffs/sgp/_C_flare.so: undefined symbol: _ZTI6Kernel
```
I think this is related to the lib linking problem, because when I check the linking, I found that:
```
$ ldd /n/home08/xiey/.conda/envs/flaretest/lib/python3.8/site-packages/flare/bffs/sgp/_C_flare.so
        linux-vdso.so.1 =>  (0x00007fff89db0000)
        libstdc++.so.6 => /n/home08/xiey/.conda/envs/flaretest/lib/python3.8/site-packages/flare/bffs/sgp/../../../../../libstdc++.so.6 (0x00002b3977922000)
        libgcc_s.so.1 => /n/home08/xiey/.conda/envs/flaretest/lib/python3.8/site-packages/flare/bffs/sgp/../../../../../libgcc_s.so.1 (0x00002b397777d000)
        libc.so.6 => /lib64/libc.so.6 (0x00002b3977ad6000)
        libm.so.6 => /lib64/libm.so.6 (0x00002b3977ea4000)
        /lib64/ld-linux-x86-64.so.2 (0x00002b39776fe000)
```
And for example, a previous working version that I compiled have all those links (there I used mkl, but in the above case I expect libopenblas instead of libmkl):
```
$ ldd /n/home08/xiey/.conda/envs/flare/lib/python3.9/site-packages/flare/bffs/sgp/_C_flare.cpython-39-x86_64-linux-gnu.so 
        linux-vdso.so.1 =>  (0x00007ffc149ad000)
        libgomp.so.1 => /n/home08/xiey/.conda/envs/flare/lib/libgomp.so.1 (0x00002b0c6a74e000)
        libpthread.so.0 => /lib64/libpthread.so.0 (0x00002b0c6ac85000)
        libmkl_intel_lp64.so => /n/home08/xiey/.conda/envs/flare/lib/libmkl_intel_lp64.so (0x00002b0c6aea1000)
        libmkl_intel_thread.so => /n/home08/xiey/.conda/envs/flare/lib/libmkl_intel_thread.so (0x00002b0c6ba40000)
        libmkl_core.so => /n/home08/xiey/.conda/envs/flare/lib/libmkl_core.so (0x00002b0c6f18f000)
        libiomp5.so => /n/sw/intel-cluster-studio-2019/lib/intel64_lin/libiomp5.so (0x00002b0c735fd000)
        libdl.so.2 => /lib64/libdl.so.2 (0x00002b0c739f2000)
        libm.so.6 => /lib64/libm.so.6 (0x00002b0c73bf6000)
        libpython3.9.so.1.0 => /n/home08/xiey/.conda/envs/flare/lib/libpython3.9.so.1.0 (0x00002b0c73ef8000)
        libstdc++.so.6 => /n/home08/xiey/.conda/envs/flare/lib/libstdc++.so.6 (0x00002b0c6a79a000)
        libgcc_s.so.1 => /n/home08/xiey/.conda/envs/flare/lib/libgcc_s.so.1 (0x00002b0c6a90f000)
        libc.so.6 => /lib64/libc.so.6 (0x00002b0c742d3000)
        librt.so.1 => /lib64/librt.so.1 (0x00002b0c746a1000)
        /lib64/ld-linux-x86-64.so.2 (0x00002b0c6a72b000)
        libutil.so.1 => /lib64/libutil.so.1 (0x00002b0c748a9000)
```
Basically, libraries like libgomp, libopenblas etc. are not linked, even though they do get installed during `conda install mir-flare`. For this issue, I don't have many ideas on this. I am also confused because looking at the build log, the [cmake output](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=601763&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642&l=352) indicates that those libs are found.
3. I add `import flare.bffs.sgp._C_flare` to the `test` in meta.yaml
4. I skipped the osx build, since the linux build is faster and practically osx is not used. But we can also recover the osx after we figured out all the issues with linux.

### CODE REVIEW

**Comment by YuuuXie** [comment]
@conda-forge-admin, please rerender
